File: Applications/Application/src/app/Model/Employee.ts
export interface Employee {
  employeeID: number;
  employeeName: string;
  gender: string;
  maritalStatus: string;
  employeeDOB: string;
  employeeAadhaarNumber: number;
  bloodGroup: string;
  employeePhoneNumber: string;
  employeeEmail: string;
  employeePassword: string;
  joiningDate: string;
  employeeRole: string;
  nationality: string;
  employeeCity: string;
  imageURL: string;
  status: boolean;
}



export interface EmployeeLogin {
  employeeEmail: string;
  employeePassword: string
}


export class EmployeeRegister {
  employeeName: string;
  employeeEmail: string;
  employeePassword: string;
  employeePhoneNumber: string;
  nationality: string;

  constructor(
    employeeName: string,
    employeeEmail: string,
    employeePassword: string,
    employeePhoneNumber: string,
    nationality: string
  ) {
    this.employeeName = employeeName;
    this.employeeEmail = employeeEmail;
    this.employeePassword = employeePassword;
    this.employeePhoneNumber = employeePhoneNumber;
    this.nationality = nationality;
  }
}

File: BackEndApplication/KanBanBoardApplication/Authentication-Application/src/main/java/com/example/AuthenticationApplication/domain/Employee.java
package com.example.AuthenticationApplication.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
@Entity
public class Employee {
    @Id
    private String employeeEmail;
    private int employeeID;
    private String employeeName;
    private String gender;
    private String maritalStatus;
    private String employeeDOB;
    private Long employeeAadhaarNumber;
    private String bloodGroup;
    private String employeePhoneNumber;
    private String employeePassword;
    private String joiningDate;
    private String employeeRole;
    private String nationality;
    private String employeeCity;
    private String status;
    private String imageURL;
}